2214/2/popular_words_counter.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'word_with_quantity'
+require_relative 'battler'
+
+class PopularWordsCounter
+  PREPOSITIONS_FILE = 'Предлоги'.freeze
+  def initialize(specific_battler, top_words)
+    @battles = specific_battler.battles
+    @top_words = top_words
+  end
+
+  def count
+    popular_words.first(@top_words).map { |popular_word| puts "#{popular_word.word} - #{popular_word.quantity} раз" }
+  end
+
+  private
+
+  def popular_words
+    usefull_words.uniq.map { |word| WordWithQuantity.new(word, usefull_words.count(word)) }.sort_by(&:quantity).reverse
```

Comment:
I like it ❤️.

I would consider moving `usefull_words.uniq`, `usefull_words.uniq.map { |word| WordWithQuantity.new(word, usefull_words.count(word)) }` to separate methods.

Read more about it: "Law of Demeter".